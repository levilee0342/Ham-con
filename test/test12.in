-22988
-26691
-7601
-8958
-2590
1996
-27105
-11804
-28283
