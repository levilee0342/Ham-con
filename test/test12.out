1996
-28283